src/mongocrypt-dll-private.h

```diff
@@ -13,7 +13,8 @@
 #define MCR_DLL_SUFFIX ".so"
 #endif
 
-#define MCR_DLL_NULL ((mcr_dll){._native_handle = NULL, .error_string = NULL})
+#define MCR_DLL_NULL \
+   ((mcr_dll){._native_handle = NULL, .error_string = MSTR_NULL})
```

Comment:
This appears to just be a whitespace change. Am I correct to assume that this won't actually address [MONGOCRYPT-389](https://jira.mongodb.org/browse/MONGOCRYPT-389) and we'll continue to see a warning when compiling this file unless we explicitly it in our compiler flags? Not a problem, as it was a minor issue to begin with, but I just want to be clear on what might be required should we need to address this on the PHPC side.

Reply:
The initializer for `error_string` was changed from `NULL` to `MSTR_NULL`. That was the cause of the warning, which wasn't obvious from the original diagnostic.

Reply:
Oh, I totally missed that while staring at the diff. Thank you!